validation/src/main/scala/hmda/validation/rules/lar/validity/V570.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Result }
+import hmda.validation.rules.EditCheck
+
+object V570 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V570"
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when(lar.lienStatus is equalTo(1)) {
+      lar.rateSpread is equalTo("NA") or {
+        try {
```

Comment:
Yeah, I meant to raise (ha) that question in the PR, but forgot to do so. Or actually a related question: why do we even have these failure messages? Are they going to be part of the API (that is, the behavior of the actual program we're writing) in any way?

For now I will assume it's worth keeping them, and yeah, I'll clean up the code.  (And honestly, the fact that you're making the same comment twice, makes it pretty clear that I also need to remove more duplication.  I was going to postpone, but if I'm cleaning up anyway, might as well go ahead.)